docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -2920,12 +2920,48 @@ and [an explicit name for the stylesheet](https://github.com/garronej/tss-react#
  export default MyCustomButton;
 ```
 
-> **Note:** `tss-react` is **not maintained** by MUI.  
-> If you have any question about how to setup SSR (Next.js) or if you are wondering
-> how to customize the `theme` object please refer to `tss-react`'s documentation,
-> the [Mui integration section](https://github.com/garronej/tss-react#mui-integration) in particular.  
-> You can also [submit an issue](https://github.com/garronej/tss-react/issues/new) for any bug or
-> feature request and [start a discussion](https://github.com/garronej/tss-react/discussions) if you need help.
+#### Overriding styles - `classes` prop
+
+[Documentation of the feature in v4](https://v4.mui.com/styles/advanced/#makestyles) - [Equivalent in `tss-react`](https://v4.mui.com/styles/advanced/#makestyles)
+
+```diff
+-import { makeStyles } from '@material-ui/core/styles';
++import { makeStyles } from 'tss-react/mui';
++import { useMergedClasses } from 'tss-react';
+
+-const useStyles = makeStyles({
++const useStyles = makeStyles()({
+  root: {}, // a style rule
+  label: {}, // a nested style rule
+});
+
+function Nested(props) {
+- const classes = useStyles(props);
++ let { classes } = useStyles();
++ classes = useMergedClasses(classes, props.classes);
+
+  return (
+    <button className={classes.root}>
+      <span className={classes.label}> // 'tss-xxxx-label my-label'
+        nested
+      </span>
+    </button>
+  );
+}
+
+function Parent() {
+  return <Nested classes={{ label: 'my-label' }} />
+}
+```
+
+#### Other questions?
+
+**Note:** `tss-react` is **not maintained** by MUI.  
```

Comment:
```suggestion
**Note:** `tss-react` is **not maintained** by MUI. 

```

Reply:
I think `#### Other questions?` is redundant.

Reply:
Yes, it is.
However, I want people to be aware that there are some extra steps for SSR.  
I was afraid that this notice would be ignored because it would appear as being part of the `#### Overriding styles - `classes` prop` section.  
That said, I understand why you prefer removing it. Your call, give me a 👍 and I'll accept your suggestion.

Reply:
👍 I don't think the notice will be ignored because you have mentioned it in the next line. I might be wrong but let's remove it and see the feedback.